benchmarks/src/main/java/org/apache/druid/benchmark/FilterPartitionBenchmark.java

```diff
@@ -149,9 +149,7 @@ public void setup() throws IOException
   {
     log.info("SETUP CALLED AT " + System.currentTimeMillis());
 
-    if (ComplexMetrics.getSerdeForType("hyperUnique") == null) {
-      ComplexMetrics.registerSerde("hyperUnique", new HyperUniquesSerde(HyperLogLogHash.getDefault()));
-    }
+    ComplexMetrics.registerSerde("hyperUnique", () -> new HyperUniquesSerde(HyperLogLogHash.getDefault()));
```

Comment:
I would create a constant for `hyperUnique` to avoid typo's. Similar to the `THETA_SKETCH`.

Reply:
@Fokko I don't think this is a problem so I merged this PR, but feel free to open a separate issue about this if you feel this is a problem.